src/main/java/step4/view/ResultView.java

```diff
@@ -0,0 +1,25 @@
+package step4.view;
+
+import step4.domain.Car;
```

Comment:
```suggestion
package step4.view;

import step4.dto.ResultData;
```

domain.Car 와는 별개로, 순수하게 데이터를 담아서 get 만 할 수 있는 Data 클래스를 하나 더 만들면 어떨까요?
제 요상한 집착일 수도 있지만, view 에서 domain 에 대한 의존성이 보이면 모델과 뷰가 제대로 분리되지 않은 느낌이 들더라고요.